configs/components/fesom/ESM_auto_tripy.yaml

```
@@ -0,0 +1,382 @@
+#___i1lvl Parameter___
+tripyrun_name: "${output_file} "
+do_papermill: True
+do_parallel: false
+input_paths:
+    - ${outdata_path}
+input_names:
+    - "${output_file}"
+   
+    
+#ref_path: /albedo/work/projects/p_fesom/pscholz/project_TRR181/trr181_tke_ctrl_ck0.1/5/
+#ref_name: $TKE$' 
+    
+mesh_path: ${mesh_dir}
+save_path: "${base_dir}/${expid}/analysis/Tripyview"
+
+which_clim: 'phc3'
+clim_path: '/work/ab0246/a270092/input/fesom2/hydrography/phc3.0/phc3.0_annual.nc'
```

Not sure about this one, it is a hard-coded path from a specific user on Levante. Maybe this should instead be something like:

```
```suggestion
clim_path: '${fesom.pool}/hydrography/phc3.0/phc3.0_annual.nc'
```

Not sure about the exact path, but something in the fesom configuration...

---

correct for awicm3 on levante